crates/interpreter/src/gas/calc.rs

```diff
@@ -344,7 +375,8 @@ pub fn memory_gas(a: usize) -> u64 {
 
 /// Initial gas that is deducted for transaction to be included.
 /// Initial gas contains initial stipend gas, gas for access list and input data.
-pub fn validate_initial_tx_gas<SPEC: Spec>(
+pub fn validate_initial_tx_gas(
```

Comment:
```suggestion
pub const fn validate_initial_tx_gas(
```

Reply:
Can't, U256 is missing a lot of const ops, most importantly Eq

Reply:
So there is not a lot of point in half changing this, as we would have two types of function?

Reply:
There already is, dynamic and static gas, I tried to make all functions I could const

Reply:
And this would be the third, but lets go with the change `const fn` are better in general.